iati_datastore/iatilib/console.py

```diff
@@ -36,9 +36,9 @@ def download_codelists():
             print "Downloading %s.xx %s" % (major_version, name)
             resp = requests.get(url[major_version])
             resp.raise_for_status()
-            assert len(resp.text) > 0, "Response is empty"
+            assert len(resp.content) > 0, "Response is empty"
```

Comment:
What's the thinking on using `.content` over `.text` ? Did you find there were issues with how `requests` attempted to decode responses?

Reply:
Relevant commit message (57356e1b) says: “I won’t pretend to understand why this is right.”

Given the datastore is due to be replaced anyway, I guess it’s not worth figuring out what this was about.

Reply:
> Did you find there were issues with how `requests` attempted to decode responses?

Yeah – so the server doesn’t specify an encoding, so `requests` decodes as ISO-8859-1 (since this is default according to RFC 2616). That’s fine until the text is output as UTF-8. The text just needs to be output using the same encoding it is read as.